.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,127 @@
+name: Bug report
+description: Create a report to help us fix something that isn't working as expected
+labels: "bug :bug:"
+body:
+- type: markdown
+  attributes: 
+    value: |
+      ## Before you begin
+      🚨 **Please do not skip instructions**🚨
+      
+      This info is essential for investigating your bug report. Issues with missing information may be closed without investigation.
+      
+      If you're uncertain about the problem, find or start a [Questions & Help](https://github.com/CommunityToolkit/WindowsCommunityToolkit/discussions/categories/questions-help) discussion, where you can get insight from the community regarding your issue. 
+      
+- type: textarea
+  id: description
+  attributes:
+     label: Describe the bug
+     description: A clear and concise description of what the bug is.
+- type: input
+  id: last-working-toolkit-version
+  attributes:
+      label: Regression
+      description: If this is a regression, what toolkit version did you last see it work?
+      placeholder: 
+  validations:
+      required: false
+- type: textarea
+  id: repro-steps
+  validations:
+    required: true
+  attributes:
+     label: Steps to reproduce
+     description: Provide steps to reproduce the issue, or let us know why it can't be reproduced (e.g. more complex setup, environment, dependencies, etc...)
+     placeholder: |
+        A repro from the sample app expidites the process of discovering, resolving, and validating bug fixes. 
+        Provide code-snippets or XAML snippets as appropriate.
+
+        Example repro steps:
+        1. Given the following environment (Sample App w/ XAML, Project with Isolated setup, etc...)
+        2. Go to '...'
+        3. Click on '....'
+        4. Scroll down to '....'
+        5. See error
+- type: textarea
+  id: expected-behavior
+  validations:
+    required: true
+  attributes:
+     label: Expected behavior
+     description: A clear and concise description of what you expected to happen.
+- type: textarea
+  id: screenshots
+  attributes:
+     label: Screenshots
+     description: If applicable, add screenshots to help explain your problem.
+- type: checkboxes
+  id: environment-windows-build
+  attributes:
+     label: Windows Build Number
+     description: Check one or more of the following options
+     options:
+       - label: Fall Creators Update (16299)
+       - label: April 2018 Update (17134)
```

Comment:
We can cut the older versions as we go 1809+ now. We should probably add the newer SDKs numbers as well.